gtsam/hybrid/HybridGaussianConditional.cpp

```diff
@@ -51,14 +51,14 @@ HybridGaussianConditional::HybridGaussianConditional(
                  discreteParents, GetFactorValuePairs(conditionals)),
       BaseConditional(continuousFrontals.size()),
       conditionals_(conditionals) {
-  // Calculate logConstant_ as the minimum of the log normalizers of the
-  // conditionals, by visiting the decision tree:
+  // Calculate logConstant_ as the minimum of the negative-log normalizers of
+  // the conditionals, by visiting the decision tree:
   logConstant_ = std::numeric_limits<double>::infinity();
   conditionals_.visit(
       [this](const GaussianConditional::shared_ptr &conditional) {
         if (conditional) {
-          this->logConstant_ = std::min(
-              this->logConstant_, -conditional->logNormalizationConstant());
+          this->logConstant_ =
+              std::min(this->logConstant_, conditional->negLogConstant());
```

Comment:
Header says max but this is still min. If this is a bug and it should be max, there is a (non-regression) unit test missing somewhere.